hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -3907,4 +3909,51 @@ public MetaRegionLocationCache getMetaRegionLocationCache() {
   public RSGroupInfoManager getRSGroupInfoManager() {
     return rsGroupInfoManager;
   }
+
+  /**
+   * Get the compaction state of the table
+   *
+   * @param tableName The table name
+   * @return CompactionState Compaction state of the table
+   */
+  public org.apache.hadoop.hbase.client.CompactionState getCompactionState(
```

Comment:
Why full class name here?

Reply:
Conflict with org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos.GetRegionInfoResponse.CompactionState

Reply:
Then better not import org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos.GetRegionInfoResponse.CompactionState directly? Import org.apache.hadoop.hbase.shaded.protobuf.generated.AdminProtos.GetRegionInfoResponse, and then use GetRegionInfoResponse.CompactionState to reference the protobuf message.

Reply:
Good idea, will fix, thanks.